File: 0236A.Boy.or.Girl/solution.py
def solution():
    have_runes = [0 for _ in range(26)]
    for rune in input():
        have_runes[ord(rune) - ord('a')] = 1

    if sum(have_runes) % 2 == 0:
        print("CHAT WITH HER!")
    else:
        print("IGNORE HIM!")


if __name__ == "__main__":
    solution()

File: 0236A.Boy.or.Girl/solution.cpp
#include <iostream>
#include <string>
#include <numeric>


void solution() {
    int have_runes[26] = {0};

    std::string input;
    std::cin >> input;

    for (const char c: input) {
        have_runes[c - 'a'] = 1;
    }

    int sum = std::accumulate(have_runes, have_runes + sizeof(have_runes) / sizeof(int), 0);
    if (sum % 2 == 0) {
        std::cout << "CHAT WITH HER!\n";
    } else {
        std::cout << "IGNORE HIM!\n";
    }
}


void setup() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(NULL);
}


int main() {
    setup();
    solution();
}
